packages/client/src/arpa_reporter/views/LoginView.vue

```diff
@@ -16,7 +16,7 @@
       </div>
       <div class="form-group">
         <button
-          class="btn btn-primary"
+          class="btn usdr-btn-primary"
```

Comment:
Note to reviewer: screenshots

## Before
- Color Contrast
- Missing H1 heading

![image](https://github.com/user-attachments/assets/8abe606f-504d-46b1-a31c-82b76d3719f8)

![image](https://github.com/user-attachments/assets/92536502-a58d-4fc6-9686-c525b2c4a42c)


## After
![image](https://github.com/user-attachments/assets/95d8d9ca-c3fc-4d6c-9cb4-849f3841d3e2)